core/src/main/kotlin/model/Documentable.kt

```diff
@@ -366,6 +389,12 @@ enum class FunctionModifiers {
     NONE, FUNCTION, EXTENSION
 }
 
+fun Variance<TypeParameter>.withNewDri(dri: DRI) = when(this) {
```

Comment:
NIT: I think you can easily drop the `new` part. `withDRI` or `withDri` will be perfectly understandable IMHO